docs/modules/operation/pages/alarms/alarm-advanced-search.adoc

```diff
@@ -0,0 +1,53 @@
+
+[[ga-advanced-alarm-search]]
+= Advanced Alarm Search
+
+Often users wish to query the alarm list with more customizations to narrow down the search.
+The Horizon alarm list page has a button to filter your searches based on different criteria.
+
+The following are the criteria which can be searched upon:
+
+* Alarm Text Contains
+** Performs a substring filter on both the log message and the description
+** To negate this search, prefix the inputted text with `-`.
+This will return results which do not contain the provided input text.
+
+* TCP/IP Address Like
+** Performs an `IPLIKE` on the provided IP Address to filter the results.
+** To negate this search, prefix the inputted text with `-`.
+This will return results which does not satisfy the `IPLIKE` function.
+
+* Node Label Contains
+** Performs a substring filter on the Node Label.
+** To negate this search, prefix the inputted text with `-`.
+This will return results which does not contain the provided input text.
+
+* Severity
+** Filter the alarms list based on the alarm severity.
+** To negate this search, toggle the checkbox below titled "Negate Severity".
+This will return results which does not match the set Severity.
```

Comment:
And to clarify ... is there a distinction between negate and exclude? In some situations, it seems that "exclude" makes more sense.

Reply:
IMO, I agree that "Exclude" may be a better verb than "negate".  Also agree that if you have a multi-select of values, there doesn't need to be a negate/exclude option as the user can pick or not pick the values as desired.

Reply:
For a Severity search that only gives one option, instead of having a checkbox to negate the single severity, there should be options for "=, <, >, <=. >=. !=" to find greater/less than as well.

Reply:
Sure, will look at making Severity a multi-select instead of a dropdown as there is multiple fixed values that could be.

Reply:
Also renaming of Negate to Exclude